src/main/java/se/kth/webtex/WebTex.java

```diff
@@ -156,11 +154,9 @@ private int getResolution(HttpServletRequest request) {
         }
     }
 
-    private String encodeURIComponent (String unencoded) {
+    private String encodeURIComponent(String unencoded) {
         try {
-            String escaped = unencoded.replace("\\",  "\\\\")
-                    .replaceAll("(\\r|\\n)+", " ")
-                    .replace("'", "\\'");
+            String escaped = unencoded.replace("\\", "\\\\").replaceAll("(\\r|\\n)+", " ").replace("'", "\\'");
```

Comment:
Strange to make this a (too) long line while making lots of other lines shorter ...